packages/neuron-wallet/src/models/multisig.ts

```diff
@@ -9,9 +11,11 @@ export interface MultisigPrefix {
   N: string
 }
 
+const BLAKE160_HEX_LENGTH = 42
```

Comment:
`BLAKE160_HEX_LENGTH` can be exported in `./blake2b`

Reply:
OK. Done!